.circleci/config.yml

```diff
@@ -110,7 +131,9 @@ commands:
             esac
             echo "Setting deployment ENVIRONMENT to $DEPLOY_ENV"
             echo "export ENVIRONMENT=$DEPLOY_ENV" >> $BASH_ENV
-            source $BASH_ENV          
+            # to install a new version of tcell, do  "tar -cvfz tcell-x.y.z.tar.gz tcell" after expanding tcell's zip
+            echo 'export TCELL_PATH=gs://ddp-tcell/tcell-1.10.0.tar.gz' >> $BASH_ENV
```

Comment:
Let's drop this in favor of YAML reference that has string and command parameter that refers to the reference. Keeps the file path to something akin to a static header declaration.

Reply:
Looks like you can get rid of this export now.